apps/app/src/test/resources/include/svs/_sv.conf

```diff
@@ -41,6 +41,8 @@
       "org.lfdecentralizedtrust.splice.sv.automation.delegatebased.ExpiredLockedAmuletTrigger",
       "org.lfdecentralizedtrust.splice.sv.automation.delegatebased.ExpiredAnsSubscriptionTrigger"
     ]
+    # example of configuration to ignore contracts for expiry
```

Comment:
what do you think of making the config a mapping from trigger name => list of contract ids? I suspect we'll need that for more triggers.

Reply:
Do you have in mind a way to easily generalize this? The way we use contract ids in the input of the triggers is really trigger specific. The way I see it we would need to do a big refactoring of the triggers to make it generic.